src/widgets/table/table.rs

```diff
@@ -799,6 +821,19 @@ impl<'a> Styled for Table<'a> {
     }
 }
 
+impl<'a, Item> FromIterator<Item> for Table<'a>
+where
+    Item: Into<Row<'a>>,
+{
+    fn from_iter<Iter: IntoIterator<Item = Item>>(rows: Iter) -> Self {
+        // When collecting from an iterator into a table, the user
+        // must provide the widths using `Table::widths` after
+        // construction.
```

Comment:
This is great, what about moving it as doc comment of the `from_iter` function?

Reply:
Also - wrap at 100 chars please for consistency

Reply:
Good call, and moved it to the docstring, wrapped at 100 chars.